nengo/spa/spa.py

```diff
@@ -88,7 +88,11 @@ def __setattr__(self, key, value):
 
     def get_module(self, name):
         """Return the module for the given name."""
-        return self._modules[name]
+        if name in self._modules:
+            return self._modules[name]
+        elif '_' in name:
+            module, name = name.rsplit('_', 1)
```

Comment:
Yes, it would work, but it would require adding that functionality to the spa/action_objects system, and that's well outside the scope of this PR.

Note: this PR is about fixing a bug in the spa thalamus system that failed on ```module_whatever``` situations, which is what the rest of the spa system was built with.  It'd be a separate PR to go change everything over to ```module.whatever``` instead of ```module_whatever```.